.github/workflows/test.yml

```
@@ -0,0 +1,34 @@
+      - yuseok
+      - wonseok
+
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [17]
+    outputs:
+        test_status: ${{ steps.test_status.outputs.status }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)